chrome/skin/style.css

```diff
@@ -116,17 +114,33 @@ popupset#mainPopupSet panel#PopupAutoCompleteRichResult richlistbox {
   min-width: 0;
 }
 
+#universal-search-recommendation-details,
+#universal-search-recommendation-description {
+  color: GrayText;
+  width:100%;
+}
+
 #universal-search-recommendation-title,
 #universal-search-recommendation-url {
   color: -moz-FieldText;
   color: FieldText;
-  display: block;
-  margin: 0;
+/*  margin: 0; */
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
 }
 
+#universal-search-recommendation-details,
+#universal-search-recommendation-description {
+  color: GrayText;
+  -moz-box-align: start;
+  text-overflow: ellipsis;
+}
+
+#universal-search-recommendation-details {
+  width: 100%;
+}
```

Comment:
Nope, messing with this breaks stuff. Just leave it as it is, as much as possible.